### PR TITLE
Use crates from RustCrypto organization and remove `rust-crypto` crate.

### PR DESCRIPTION
**Pull request by jeandudey**

### CODE REVIEW

File: crypto/src/lib.rs [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
-use rcrypto::ripemd160::Ripemd160;
+use digest::generic_array::GenericArray;
+use digest::generic_array::typenum::{U20, U32, U64};
+use sha1::Sha1;
+use sha2::Sha256;
+use ripemd160::Ripemd160;
 use siphasher::sip::SipHasher24;
 use primitives::hash::{H32, H160, H256};
 
+pub use digest::Digest;
```
Comment (svyatonik) [inline crypto/src/lib.rs]: Seems like `pub use` is not really required here.

Comment (jeandudey) [reply crypto/src/lib.rs]: @svyatonik chain depends on `Digest` being re-exported by `bitcrypto`, also it's a common pattern in RustCrypto crates to re-export the `Digest` trait.

**Comment by jeandudey** [comment]
@mkjekk I'm sorry, what's that?

**Comment by jeandudey** [comment]
Closing due to lost of interest

**Pull request closed without merging**